File: TransportationService.Api/Models/TransportOrder.cs
namespace TransportationService.Api.Models;

public class TransportOrder
{
    public Guid Id { get; set; }
    public string Reference { get; set; } = string.Empty;
    public string CustomerName { get; set; } = string.Empty;
    public string PickupAddress { get; set; } = string.Empty;
    public string DeliveryAddress { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
}

File: TransportationService.Web/src/features/transport-orders/types/transportOrder.ts
export interface TransportOrder {
  id: string
  reference: string
  customerName: string
  pickupAddress: string
  deliveryAddress: string
  status: string
}

export interface CreateTransportOrderInput {
  reference: string
  customerName: string
  pickupAddress: string
  deliveryAddress: string
  status: string
}

export const TRANSPORT_ORDER_STATUS_OPTIONS: readonly string[] = [
  'New',
  'Planned',
  'InProgress',
  'Delivered',
  'Cancelled',
]
